Vagrantfile

```diff
@@ -128,3 +132,4 @@ Vagrant.configure("2") do |config|
     end
   end
 end
+
```

Comment:
We could make this a `{host_path => guest_path}` hash instead?

On Wed, Jan 21, 2015 at 5:04 PM, Brian Waldon <notifications@github.com>
wrote:

> In Vagrantfile
> <https://github.com/coreos/coreos-vagrant/pull/198#discussion_r23348744>:
>
> > @@ -128,3 +132,4 @@ Vagrant.configure("2") do |config|
> >      end
> >    end
> >  end
> > +
>
> Unnecessary newline.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/coreos-vagrant/pull/198/files#r23348744>.
>